src/rosecommon/src/itemdb.cpp

```diff
@@ -0,0 +1,60 @@
+#include "itemdb.h"
+#include "connection.h"
+
+using namespace RoseCommon;
+
+ItemDatabase::ItemDatabase() {
+  for ( int index = 1; index < 15; ++index ) {
```

Comment:
15 would be nice to have as a constant somewhere, so it doesn't become a magic number later on

Reply:
well 15 isn't a magic number and it isn't a hard limit. It's only here to ensure we reserve the first 15. You can have more then 15 indexes.

Reply:
But still, it's better to have it declared once somewhere than having to copy paste it around. You already use it twice in the ItemDatabase code, I think it's better to have it defined in the class itself.